scripts/wix.ts

```diff
@@ -0,0 +1,19 @@
+// This script builds the wix installer, assuming the zip file has already been
+// built (and dist/win-unpacked is populated).
+// This is only used during development.
+
+import path from 'path';
+
+import buildInstaller from './lib/installer-win32';
+
+async function run() {
+  const distDir = path.join(process.cwd(), 'dist');
+  const appDir = path.join(distDir, 'win-unpacked');
+
+  await buildInstaller(distDir, appDir);
+}
+
+run().catch((ex) => {
```

Comment:
This script should probably verify that it's being run on a Windows box and that `dist/win-unpacked/resources/app.asar` exists, with a friendly reminder to first run `npm run build` if it doesn't.

Reply:
Added a check for the file in 197372ea93ae0559f39da11dcdc16447413f1e8a. (No need to check that it's running on Windows, because nobody else would have `win-unpacked`.)